src/helpers/findAll.ts

```diff
@@ -17,7 +19,7 @@ export function findAll(
   root: ReactTestInstance,
   predicate: (element: ReactTestInstance) => boolean,
   options?: FindAllOptions
-) {
+): Array<HostReactTestInstance> {
```

Comment:
[nit] `HostReactTestInstance[]`

Reply:
I personally prefer the Array syntax because the the first thing you see is that it's an array, whereas I didn't see the [] at the end multiple times in the past. Also we say "an array of number" so it feels more natural to me. That being said it's better to have consistency across the codebase so I'll make the change

Reply:
As I wrote, this is nit, so let's keep it as is.